### PR TITLE
chore: rename cypress happo project in workflows

### PR DESCRIPTION
**Pull request by ozgurkececioglu**
[FX-2874]

### Description

- rename `picasso` project name to `Picasso/Cypress` inside our workflows. Main reason for that is to eliminate confusion with `Picasso/Storybook` project

HAPPO has a fallback mechanism for its configuration settings. I added `picasso` to the alternative names field so it will still catch the correct project for ongoing CI processes.
Screen Shot 2022-06-21 at 13 12 57

### How to test

- there shouldn't be a problem with our CI for cypress

### Development checks

- Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- Annotate all `props` in component with documentation
- Create `examples` for component
- Ensure that deployed demo has expected results and good examples
- Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- Covered with tests

**Breaking change**

- codemod is created and showcased in the changeset
- test alpha package of Picasso in StaffPortal

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2874]: https://toptal-core.atlassian.net/browse/FX-2874?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by separatio — APPROVED** [review]
Overall it looks good, but I have 2 questions. Will a new project be created on Happo's side? How will they know that we are just renaming it?

**Comment by ozgurkececioglu** [comment]
> Overall it looks good, but I have 2 questions. Will a new project be created on Happo's side? How will they know that we are just renaming it?

I have already replied to your question in the description of the PR. I have already renamed the project on Happo side.

**Comment by separatio** [comment]
Yeah. I jumped into the code and didn't look at the description. :facepalm: I'll try to be more careful next time, not ask already answered questions. Thanks Oz! :bow:

**Review by separatio — COMMENTED** [review]
There's one more thing you need to change, though. Required checks for protected master branch, after this PR is merged.

image

**Comment by ozgurkececioglu** [comment]
> There's one more thing you need to change, though. Required checks. I am already getting an error on the TreeView PR.
> 
> image

Hmm, in this case, we should merge this one fast to not block others 🤔 I wasn't aware of that

**Review by TomasSlama — APPROVED** [review]

File: .github/workflows/davinci-integration-tests.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ on:
         required: true
       NPM_TOKEN:
         required: true
+      HAPPO_PROJECT:
+        required: true
```
[the remark this answers is not in the document]
Comment (dmaklygin) [reply .github/workflows/davinci-integration-tests.yml:11]: you can remove it :)

**Review by OndrejTuma — APPROVED** [review]
LGTM 👍

**Pull request merged**